scripts/convert_scss_to_css.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
**Tip**: _"I don't know if this applies as good practice"_
Consider removing this file from git tracker because it only applies to your machine.